File: Aula_1/Carro.java
package lpoo2;


public class Carro {
    public String modelo;
    public int ano;
    void ligar() {
        System.out.println("Carro Ligado!");
    }
    
    public static void main(String[] args) {
        Carro C = new Carro();
        C.modelo = "Polo";
        C.ano = 2005;
        C.ligar();
    }
}

File: Aula_1/Carro.cpp
#include <iostream>
#include <string>

using namespace std;
class Carro
{
public:
    string modelo;
    int ano;
    void ligar()
    {
        cout << "Carro Ligado!" << endl;
    }
};

int main()
{
    Carro C;
    C.modelo = "Polo";
    C.ano = 2005;
    C.ligar();
    return 0;
}
